application.py

```diff
@@ -28,7 +28,9 @@ def configure_logging():
         'DEBUG': logging.DEBUG,
     }
     handler = logging.StreamHandler()
-    logging.basicConfig(level=levels[EQ_LOG_LEVEL], format=log_format, handlers=[handler])
+    logging.basicConfig(
```

Comment:
If you have `black`'d this then you should probably update the format stop to include this.